commands/dev-deploy.js

```diff
@@ -36,6 +36,10 @@ module.exports = {
 };
 
 async function devDeploy(options) {
+    const currentNetwork = options.networkId;
```

Comment:
Better test for mainnet specifically, betanet should support dev-deploy for example

Reply:
@vgrichina
Yes, I thought the same, but it does not work on `betanet`, and `dev-deploy` docs are explicitly targeting `testnet`.

Reply:
@volovyk-s why it doesn't work on betanet, what does happen?

Reply:
@vgrichina It throws an error:
`BadRequestError: Error: {"index":0,"account_id":"dev-1618909283091-6810991","predecessor_id":"betanet","registrar_account_id":"near","kind":{"account_id":"dev-1618909283091-6810991","registrar_account_id":"near","predecessor_id":"betanet"}}
`

Reply:
@volovyk-s so this seems a betanet helper misconfiguration issue that we need to fix

Reply:
Seems to work fine for me when sending request directly:

```
➜  near-contract-helper git:(master)  http post https://helper.betanet.near.org/account newAccountId="test1.betanet" newAccountPublicKey="ed25519:JCdUG8doFbVHTCtxuy2RVXraHg8NRhxdCcpiyGgcBMAh"
HTTP/1.1 200 OK
Content-Encoding: gzip
Content-Length: 748
Content-Type: application/json; charset=utf-8
Date: Wed, 21 Apr 2021 05:27:58 GMT
Server: Render
Vary: Accept-Encoding
X-Ratelimit-Limit: 10
X-Ratelimit-Remaining: 9
X-Ratelimit-Reset: 1618983776.276166

{
    "receipts_outcome": [
        {
            "block_hash": "CgHd9r2d4nkMaDWfQLNzsZe2rpxGJCz9MpTVyKQTPwoq",
            "id": "8ffLR9kvx9bGjyhbHw7ogqoVwjsngXxmFaXnztCeWJwE",
            "outcome": {
                "executor_id": "test1.betanet",
                "gas_burnt": 937144500000,
                "logs": [],
                "receipt_ids": [
                    "EaeLPpkGyicTNzE5Yy3WBXfYBCXDJUXvn2gueBsYDSno"
                ],
                "status": {
                    "SuccessValue": ""
                },
                "tokens_burnt": "93714450000000000"
            },
            "proof": []
        },
        {
            "block_hash": "EQcvbmkFtDFRwuPZdrAqGHX7946ti1CDvwuGCsPi6tSu",
            "id": "EaeLPpkGyicTNzE5Yy3WBXfYBCXDJUXvn2gueBsYDSno",
            "outcome": {
                "executor_id": "betanet",
                "gas_burnt": 0,
                "logs": [],
                "receipt_ids": [],
                "status": {
                    "SuccessValue": ""
                },
                "tokens_burnt": "0"
            },
            "proof": []
        }
    ],
    "status": {
        "SuccessValue": ""
    },
    "transaction": {
        "actions": [
            "CreateAccount",
            {
                "Transfer": {
                    "deposit": "100000001000000000000000000"
                }
            },
            {
                "AddKey": {
                    "access_key": {
                        "nonce": 0,
                        "permission": "FullAccess"
                    },
                    "public_key": "ed25519:JCdUG8doFbVHTCtxuy2RVXraHg8NRhxdCcpiyGgcBMAh"
                }
            }
        ],
        "hash": "6YPEfWVpfmq42HeTEEyhTqDLAHy8wxYUsghq7s8LQkVc",
        "nonce": 278,
        "public_key": "ed25519:9r1FcBRdop8NeQtiFWEWHNWtDWhTdAe2YvU8h7FrW53x",
        "receiver_id": "test1.betanet",
        "signature": "ed25519:3bb35VHqbrSyYy2Hf6xwKMreQ8Fmdh96E5Er7viaK8DCTi3SohZCE2TtJ8YQKGUi4qFnutBN6RwSY7Sech98QCm1",
        "signer_id": "betanet"
    },
    "transaction_outcome": {
        "block_hash": "An99mcWwbrUZ3VW2sJod7ZuWvmz7BJbrbnFzN6fMdgNi",
        "id": "6YPEfWVpfmq42HeTEEyhTqDLAHy8wxYUsghq7s8LQkVc",
        "outcome": {
            "executor_id": "betanet",
            "gas_burnt": 937144500000,
            "logs": [],
            "receipt_ids": [
                "8ffLR9kvx9bGjyhbHw7ogqoVwjsngXxmFaXnztCeWJwE"
            ],
            "status": {
                "SuccessReceiptId": "8ffLR9kvx9bGjyhbHw7ogqoVwjsngXxmFaXnztCeWJwE"
            },
            "tokens_burnt": "93714450000000000"
        },
        "proof": []
    }
}
```

Reply:
@vgrichina it works on `betanet` only when accountId has `.betanet` suffix. `near-cli` creates acc in a next way: `accountId = dev-${Date.now()}-${randomNumber}`. Without any suffix. Works fine for `testnet` but not for `betanet`.

Reply:
That is odd! Is testnet still misconfigured allowing non-suffixed account ids? cc @bowenwang1996

Reply:
@frol why misconfigured? AFAIK even mainnet allows to create arbitrary account IDs if they are at least 32 characters long.

For example https://explorer.near.org/accounts/mikemikemikemikemikemikemikemike

Reply:
@frol where this configuration is? Who is responsible for it?

Reply:
Oh, I thought the account id is going to be shorter than 32 characters, so it depends on how big the `randomNumber` is then.

Reply:
@vgrichina @frol I have increased dev acc length.